docs/_docs/Users/build.md

```diff
@@ -105,6 +146,18 @@ bash ./phoenicis-dist/phoenicis.sh
 
 ## Troubleshooting
 
+### Maven dependencies cannot be downloaded
```

Comment:
As a short note:
Sometimes it also helps to delete the `.m2` folder in your home folder.
But this is a drastic measure, which basically resets maven and forces it to redownload all dependencies.